SPECS/lcov/lcov.spec

```diff
@@ -0,0 +1,144 @@
+Name: lcov
+Version: 1.14
+Release: 6%{?dist}
+Summary: LTP GCOV extension code coverage tool
+License: GPLv2+
+
+URL: https://github.com/linux-test-project/lcov/
```

Comment:
Please lint all new specs - it'll put the tags in the right order, fix paths, etc.